DevVehicle35-Motors/App/Program.cs

```diff
@@ -10,6 +10,7 @@
     Console.WriteLine("Select the type of vehicle you want to build:");
     Console.WriteLine("3. Quad");
     Console.WriteLine("q. Quit");
+    Console.WriteLine("Write minibus");
```

Comment:
this option should be showed in list above.

Reply:
Yes, sorry, this line was here because we were doing a test. But now it isn´t here